### DISCUSSION TITLE
How to find and use OpenJPEG library?

### DISCUSSION THREAD

**Discussion by ghost** [JulNadeauCA/BSDBuild#15] [General]
OpenJPEG provides CMake script so it's very simple to find and use it on CMake: `find_package(OpenJPEG REQUIRED)`

https://chromium.googlesource.com/external/github.com/uclouvain/openjpeg/+/refs/tags/v2.1.1/INSTALL.md

**Comment by JulNadeauCA** [comment]
A new test module `openjpeg` was added in https://github.com/JulNadeauCA/BSDBuild/commit/f23e49abccd3893c1a215f0f5f39a9c6557beb75. It has been tested with OpenJPEG 2.5.2.
Thanks!
